### PR TITLE
Add iotime field to dio widget.

### PR DESCRIPTION
**Pull request by whirm**

### CODE REVIEW

**Comment by Mic92** [comment]
Hi, now that the freebsd pull request landed: can you rebase your patch?

**Comment by whirm** [comment]
Done!

File: widgets/dio_linux.lua [hunk 1]
```diff
@@ -49,18 +49,20 @@ local function worker(format)
...
         -- * divide by timediff because we don't know the timer value
         local read  = (stats[1] - last_stats[1]) / interval
         local write = (stats[2] - last_stats[2]) / interval
+        local iotime = (stats[3] - last_stats[3]) / interval
 
         -- Calculate and store I/O
         helpers.uformat(disk_usage, device.." read",  read,  unit)
         helpers.uformat(disk_usage, device.." write", write, unit)
         helpers.uformat(disk_usage, device.." total", read + write, unit)
+        helpers.uformat(disk_usage, device.." iotime", iotime, unit)
```
Comment (Mic92) [inline widgets/dio_linux.lua]: Ah, forgot to mention, that this field should be also documented in `Readme.md`

**Comment by whirm** [comment]
Done.

I've also made a small improvement: It doesn't make sense to have `iotime_kb` and `iotime_mb`, so I've added custom units for it.

**Comment by Mic92** [comment]
Thanks for your contribution!

**Pull request merged**